README.md

```diff
@@ -56,5 +56,96 @@ console.log(errors.AccessDenied);
 
 ```
 
+## Clustering
+
+The clustering class can be used to set up a cluster of workers. The class will
+create at least 1 worker, will log any worker event (started, exited).
+The class has also a watchdog who will restart workers until the stop() method
```

Comment:
The class **also provides a watchdog which restarts the workers in case of failure** until the stop() method